test/index_test.js

```diff
@@ -44,6 +45,12 @@ describe('maskXml()', () => {
     `);
   });
 
+  it('should skip masking non-strings', () => {
+    [undefined, { biz: 'baz' }, 42].forEach(nonString => {
+      should(maskXml(['foo'])(nonString)).equal(nonString);
```

Comment:
Use the syntax `<object>.should…` when possible please.

Reply:
That would also fail with a `TypeError: Cannot read property 'should' of undefined`